### PR TITLE
Modify libdnf5-devel to generate pkgconf(libdnf5)

### PR DESCRIPTION
**Pull request by j-mracek**
It also allow parallel installation of libdnf5-devel and libdnf-devel.

### CODE REVIEW

File: libdnf/CMakeLists.txt [hunk 1]
```diff
@@ -75,7 +75,7 @@ if(WITH_COMPS)
 endif()
 
 if (WITH_ZCHUNK)
-    pkg_check_modules(ZCHUNKLIB zck>=0.9.11 REQUIRED)
+    pkg_check_modules(ZCHUNKLIB zck>=0.9.11 REQPC_REQUIRESUIRED)
```
Comment (jan-kolarik) [inline libdnf/CMakeLists.txt:78]: This is probably a typo?

File: libdnf/libdnf5.pc.in [hunk 2]
```diff
@@ -2,7 +2,7 @@ prefix=@CMAKE_INSTALL_PREFIX@
 libdir=${prefix}/@CMAKE_INSTALL_LIBDIR@
 includedir=${prefix}/include
 
-Name: libdnf
+Name: libdnf5
```
Comment (m-blaha) [inline libdnf/libdnf5.pc.in:5]: Shouldn't also `Libs: -L${libdir} -ldnf` in this file be changed to `Libs: -L${libdir} -ldnf5`?
(Sorry, github doesn't allow me to comment on the more appropriate line #10)

File: libdnf/CMakeLists.txt (hunk 1 above)
Comment (j-mracek) [reply libdnf/CMakeLists.txt:78]: I have no idea what happened here

File: libdnf/libdnf5.pc.in (hunk 2 above)
Comment (j-mracek) [reply libdnf/libdnf5.pc.in:5]: Updated, @m-blaha May I ask you to look at the last update?

**Review by m-blaha — APPROVED** [review]

**Review by jan-kolarik — APPROVED** [review]

**Pull request merged**